packages/venia-concept/webpack.config.js

```diff
@@ -42,8 +41,8 @@ module.exports = async function(env) {
         output: {
             path: themePaths.output,
             publicPath: process.env.MAGENTO_BACKEND_PUBLIC_PATH,
-            filename: '[name].js',
-            chunkFilename: '[name]-[chunkhash].js',
+            filename: 'js/[name].js',
+            chunkFilename: 'js/[name]-[chunkhash].js',
```

Comment:
This change necessitates a corresponding change in `root.phtml`, doesn't it? The browser `404`s on these requests until I update the hardcoded `script src` in that file.

Further, I'm not sure this is the best way to do this. Couldn't we handle this with paths in `.env`?

- Expect that `MAGENTO_BACKEND_PUBLIC_PATH` end with `/js`?
- Look for two "output path" env vars—one for dev, one for prod?

Also, how is this possible? This one asset, `logo.svg`, is output to the root instead of `/js`.

<img width="286" alt="wat" src="https://user-images.githubusercontent.com/2110986/44121972-aaabbe80-9fd6-11e8-8fbf-c9b3d0c18d3f.png">

<img width="105" alt="paths" src="https://user-images.githubusercontent.com/2110986/44121997-c9aa5080-9fd6-11e8-9d9e-568795704c00.png">

Reply:
The original intent of this PR was to require that you update `.env` and add `/js` to the `MAGENTO_BACKEND_PUBLIC_PATH` var, but I forgot to include that part, which is my fault.

However, `root.phtml` should now automatically generate view URLs, which should allow us to stop using that environment variable in the first place.

At any rate, this is a large enough change that it ought to be in the 1.x release, which I'm putting together right now.